tests/test_seahash.py

```diff
@@ -29,6 +30,23 @@ def test_hashlib_compatible():
     assert s.intdigest() == int.from_bytes(s.digest(), sys.byteorder)
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 11),
+    reason="requires hashlib.file_digest which is new in python3.11",
+)
+def test_hashlib_file_digest_compatible(tmp_path):
+    import hashlib
```

Comment:
Nit: This import should live at the top of the file. I realise it's only useful for this test, but it's still easier to maintain if imports are done in 1 place.